vcpkg-configuration.json

```diff
@@ -0,0 +1,3 @@
+{
+  "overlay-ports": ["ports"]
```

Comment:
Does this override or supplement the ports in the submodule?

Reply:
Override. Anything found in an overlay ports directory is used without regard to versioning. I wanted to make sure and use our specific versions rather than updating all of our dependencies to whatever was in the vcpkg registry and this was the easiest way I could figure out how to do that.